### PR TITLE
[tests-only] Adjust requestOcsTest interactions for oCIS server

### PR DESCRIPTION
**Pull request by saw-jan**
part of https://github.com/owncloud/owncloud-sdk/issues/644 and https://github.com/owncloud/owncloud-sdk/issues/747

adjust interactions for oCIS:
- requestOcsTest.js

### CODE REVIEW

File: tests/requestsOcsTest.js [hunk 1]
```diff
@@ -60,11 +66,11 @@ describe('Main: Currently testing low level OCS', function () {
         headers: validAdminAuthHeaders
       })
       .willRespondWith({
-        status: 401,
+        status: 404,
```
Comment (Talank) [inline tests/requestsOcsTest.js:69]: shouldn't it be `401`, and the ocs status code be `997` according to the issue that you mentioned in the comment above? :thinking:

Comment (saw-jan) [reply tests/requestsOcsTest.js:69]: `401` is not expected while trying to update `non-existing` user by admin

expected for v1: http `200` and ocs `101`
expected for v2: http `404` and ocs `404`

**Review by Talank — COMMENTED** [review]
LGTM :+1:

File: tests/requestsOcsTest.js [hunk 2]
```diff
@@ -126,7 +136,6 @@ describe('Main: Currently testing low level OCS', function () {
           }
         }
       })
-    await getUserInformationAsAdminInteraction(provider)
```
Comment (jasson99) [inline tests/requestsOcsTest.js:129]: may be we don't need it in pact as well then?

[the remark this answers is not in the document]
Comment (saw-jan) [reply tests/requestsOcsTest.js:129]: yeah, this interaction is not being used anywhere.
or may be we can keep it and use it in future?

**Review by dpakach — COMMENTED** [review]
LGTM :+1:

**Review by swoichha — COMMENTED** [review]
lgtm

File: tests/requestsOcsTest.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jasson99) [reply tests/requestsOcsTest.js:129]: I would prefer to not keep the code that is not needed at the moment `You Aren’t Gonna Need It (YAGNI)`. if we need it later, we can still have the record here and can use it. But you can decide.

**Review by jasson99 — COMMENTED** [review]
I have written my opinion, you can decide on that. Other than that, It looks good to me

**Review by individual-it — APPROVED** [review]

File: tests/requestsOcsTest.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (saw-jan) [reply tests/requestsOcsTest.js:129]: convincing to me.
done deletion

**Pull request merged**